sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/EncryptionCosmosAsyncContainer.java

```diff
@@ -89,7 +92,7 @@
                 if (decryptResponse) {
                     return setByteArrayContent(rsp,
                         EncryptionProcessor.decrypt(EncryptionModelBridgeInternal.getByteArrayContent(rsp),
-                            this.encryptor).map(pair -> pair.getLeft()).publishOn(encryptionScheduler));
+                            this.encryptor).map(Pair::getLeft).publishOn(encryptionScheduler));
```

Comment:
Are these possible to put as style guides like in VS?

Reply:
I fixed them based on Intelli-J "style violations". We have a couple of style (and FxCop like) rules in the CI/Build pipeline - but mostly leave the configuration of what gets enforced to the Azure central team. 

So far on the Java side that has worked reasonably well IMO: @moderakh / @kushagraThapar - thoughts?

Reply:
We already have code style enforcer which works pretty well in IMO as well and as Fabian mentioned the code-style enforcer runs as a gated CI on every PR by the central sdk team.

As Fabian mentioned this lambda's code style is only intellj's suggestion. I wouldn't add more rules to the existing automated enforcer. being to restrictive makes it hard for coding.